src/NUnitConsole/nunit3-console/ConsoleOptions.cs

```diff
@@ -77,6 +77,18 @@ public class ConsoleOptions : CommandLineOptions
 
         #endregion
 
+        #region Overrides
+
+        protected override void CheckOptionCombinations()
+        {
+            base.CheckOptionCombinations();
+
+            if (RunAsX86 && ProcessModel == "InProcess")
+                ErrorMessages.Add("The --x86 and --inprocess options are incompatible.");
```

Comment:
Are they incompatible if you are running on an x86 machine? I agree that there is no reason to specify the x86 flag in that case, but is it invalid?